final-task/src/views/footer/footerView.ts

```diff
@@ -0,0 +1,232 @@
+import tags from '../../tags/tags';
+
+export default class FooterView {
+  private footer: HTMLElement;
+
+  private contactUs: HTMLElement;
+
+  private mainContent: HTMLElement;
+
+  private command: HTMLElement;
+
+  constructor() {
+    this.footer = tags.div(['footer']).getElement();
+    this.contactUs = tags.div(['footer__content']).getElement();
+    this.mainContent = tags.div(['footer__content']).getElement();
+    this.command = tags
+      .div(['footer__content', 'footer__command'])
+      .getElement();
+    this.footer.append(this.mainContent, this.contactUs, this.command);
+    this.createFooterBlock();
+  }
+
+  getContent() {
+    return this.footer;
+  }
+
+  private createFooterBlock() {
+    this.createHeading();
+    this.createInformation();
+    this.createFollowUs();
+    this.createContactUs();
+    this.createPayment();
+    this.createEndFooter();
+    this.createTeam();
+    this.createEmailBlock();
+  }
+
+  private createHeading() {
+    const frostGlideSvg = tags.div(['footer__rost-glide']).getElement();
+    const heading = tags.h1(['footer__h1'], 'rost glide');
+    heading.prepend(frostGlideSvg);
+    this.mainContent.append(heading);
+  }
```

Comment:
Can you please change it to match the original logo:
![image](https://github.com/ksarise/eCommerce-Application/assets/18985111/176a9a8d-db51-4b92-aee5-a7e8e86a6262)
And using this special font:
[Primal-Font.zip](https://github.com/user-attachments/files/15928258/Primal-Font.zip
But use this font only for logo title, don't change main font (!)

Reply:
fixed!